README.md

```diff
@@ -9,6 +9,9 @@ A backup application for the [Android Open Source Project](https://source.androi
 - User-friendly encryption using a mnemonic phrase (BIP39).
 - Automatic daily backups that run in the background.
 
+## Requirements
+- Android 10 or higher
```

Comment:
Maybe just "Android 10" since we may not support 11 when it is released?

Reply:
sounds reasonable yes